tests/CommandBusTest.php

```diff
@@ -67,4 +68,24 @@ public function testHandleThrowExceptionForInvalidCommand()
     {
         (new CommandBus([]))->handle('must be an object');
     }
+
+    public function testIfOneCanOnlyCreateWithValidMiddlewares()
+    {
+        $middlewareList = [$this->getMock('stdClass')];
+
+        try {
+            new CommandBus($middlewareList);
+            $this->fail('Should not be able to create a CommandBus without a valid list of middlewares!');
+        } catch (InvalidMiddlewareException $caught) {
```

Comment:
Huh, TIL.

The exception itself is already fairly meaningful so just the expectedException annotation is enough for me here. If you'd like to be more specific, either format is fine with me.